src/modules/components/proposal/proposalDataListItem/proposalDataListItemStructure/proposalDataListItemStructure.api.ts

```diff
@@ -61,7 +61,23 @@ export interface IProposalDataListItemStructureBaseProps<TType extends ProposalT
     voted?: boolean;
 }
 
-export interface IApprovalThresholdResult {
+export interface IProposalStage {
+    /**
+     * Name of the proposal stage
+     */
+    stageTitle?: string;
+
+    /**
+     * Id of the proposal stage
+     */
+    stageId?: string | number;
+}
+
+export interface IPublisher extends ICompositeAddress {
+    profileLink?: string;
```

Comment:
We can call the property just `link`. I would also a short description for the property